app/javascript/angular/code/services/_mobile_sessions.js

```diff
@@ -67,7 +67,7 @@ export const mobileSessions = (
 
 
     onSessionsFetch: function() {
-      if (!storage.isCrowdMapLayerOn()) {
+      if (!params.get('data').crowdMap) {
```

Comment:
I'd change `crowdMap` to `isCrowdMap` since it's a boolean

Reply:
I'd suggest adding something like `isCrowdMapOn` cause that is a feature that is either on or off, correct?